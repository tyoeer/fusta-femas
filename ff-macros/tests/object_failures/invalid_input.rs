use ff_macros::*;

#[derive(Object)]
enum Test {}

fn main() {}