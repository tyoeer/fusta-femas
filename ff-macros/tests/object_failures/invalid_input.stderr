error: enums aren't supported
 --> tests/object_failures/invalid_input.rs:4:1
  |
4 | enum Test {}
  | ^^^^
